packages/iperf3.rb

```diff
@@ -17,16 +17,5 @@ class Iperf3 < Package
      x86_64: '790a85d3e616b4e02b05c43c6e2ae31d0f8e100a6020f060279ba7dec427c8f5'
   })
 
-  def self.build
-    system './configure',
-           "--libdir=#{CREW_LIB_PREFIX}",
-           "--prefix=#{CREW_PREFIX}",
-           '--disable-dependency-tracking',
-           '--disable-maintainer-mode'
-    system 'make'
-  end
-
-  def self.install
-    system 'make', "DESTDIR=#{CREW_DEST_DIR}", 'install'
-  end
+  configure_options '--disable-maintainer-mode'
```

Comment:
maintainer mode is already disabled by default for iperf:
https://github.com/esnet/iperf/blob/master/configure.ac#L44C1-L44C19
https://www.gnu.org/software/automake/manual/html_node/maintainer_002dmode.html